Makefile

```diff
@@ -109,6 +110,12 @@ manifests: controller-gen
 fmt: groupimports
 	$(CGOENV) go fmt ./...
 
+gosec:
+ifeq (,$(shell which gosec))
+	curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $(shell go env GOPATH)/bin v2.2.0
```

Comment:
Hey, we manage all our external dependencies in a tools file, could you please follow this PR https://github.com/pingcap/chaos-mesh/pull/281 to update the Makefile? Thanks!